http4s-client-monix-catnap-micrometer/src/main/scala/com/avast/sst/http4s/client/monix/catnap/micrometer/Http4sClientCircuitBreakerModule.scala

```diff
@@ -0,0 +1,48 @@
+package com.avast.sst.http4s.client.monix.catnap.micrometer
+
+import cats.effect.{Clock, Resource, Sync}
+import cats.syntax.applicativeError._
+import cats.syntax.flatMap._
+import com.avast.sst.monix.catnap.CircuitBreakerMetrics.State.{Closed, HalfOpen, Open}
+import com.avast.sst.monix.catnap.micrometer.MicrometerCircuitBreakerMetrics
+import com.avast.sst.monix.catnap.{CircuitBreakerConfig, CircuitBreakerModule}
+import io.micrometer.core.instrument.MeterRegistry
+import org.http4s.Response
+import org.http4s.client.Client
+import org.slf4j.LoggerFactory
+
+object Http4sClientCircuitBreakerModule {
+
+  private lazy val logger = LoggerFactory.getLogger("Http4sClientCircuitBreakerModule")
+
+  /**  */
+  def make[F[_]: Sync](name: String,
+                       circuitBreakerConfig: CircuitBreakerConfig,
+                       client: Client[F],
+                       meterRegistry: MeterRegistry,
+                       clock: Clock[F]): Resource[F, Client[F]] = {
+    val F = Sync[F]
+
+    class ServerFailure(val response: Response[F], val close: F[Unit]) extends Exception
```

Comment:
who is the supposed receiver/consumer of this exception? it seems to me it's just for internal control flow, and thus should also be private.

Reply:
It's inside the method so it is "private".

Reply:
:+1: